src/main/kotlin/io/hemin/wien/model/Atom.kt

```diff
@@ -11,4 +14,9 @@ data class Atom(
     val authors: List<Person>,
     val contributors: List<Person>,
     val links: List<Link>
-)
+) {
+    companion object Factory : BuilderFactory<Atom, AtomBuilder> {
+        /** Returns a builder implementation for building [Atom] model instances. */
+        @JvmStatic override fun builder(): AtomBuilder = ValidatingAtomBuilder()
```

Comment:
Maybe a nitpick, but according to the [Kotlin style guide](https://kotlinlang.org/docs/reference/coding-conventions.html#annotation-formatting), the annotation should likely be on its own line (here and elsewhere)

Reply:
Actually I had a look at the style guide when I wrote this. I saw the inline notation while researching the correct annotation to use here, and the style guide's third example confirm this usage, so I stuck with it.

But I am happy to use the separate line notation however. I'm more used to it from Java anyway.

Reply:
Ah, didn't notice it. Personal taste: let's be consistent :) The only cases where I am happy to put it on the same line:

```kotlin
data class Book(
    @SerializedName("id") bookId: String
)
```

:)